libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -833,6 +833,13 @@ static DNNReturnType execute_model_tf(const DNNModel *model, const char *input_n
             out_frame->height = outputs[0].height;
         }
         break;
+    case DFT_ANALYTICS_DETECT:
+        if (!model->detect_post_proc) {
+            av_log(ctx, AV_LOG_ERROR, "Detect filter needs provide post proc\n");
+            return DNN_ERROR;
+        }
+        model->detect_post_proc(out_frame, outputs, nb_output, model->filter_ctx);
+        break;
```

Comment:
this patch is for vf_dnn_detect, why we change in dnn/dnn_backend_tf.c?  such change can be merged in previous patch.

Reply:
you mean move this part to 'support detect model' patch?

Reply:
yes